proxy/envoy/policy.go

```diff
@@ -61,6 +61,7 @@ func insertMixerFilter(listeners []*Listener, instances []*model.ServiceInstance
 							"source.ip":  context.IPAddress,
 							"source.uid": context.UID,
 						},
+						QuotaName: "RequestCount",
```

Comment:
Internally we have a static map of `method -> quota bucket` so that the proxy knows what to check against. I don't see why we couldn't use that model.

Perhaps more naturally in mixer-land, we could allow the user to write a set of expressions we evaluate to determine the quota bucket. Then we remove the quota bucket name from the `Quota` API's request object. I like this model more, personally.